editor/src/clj/internal/transaction.clj

```diff
@@ -410,7 +432,7 @@
     (as-> ctx ctx'
       (apply-tx ctx' (concat new-override-nodes-tx-data
                              new-override-tx-data))
-      (apply-tx ctx' (init-fn (in/custom-evaluation-context {:basis (:basis ctx')})
+      (apply-tx ctx' (init-fn (in/custom-evaluation-context {:basis (:basis ctx') :tx-data-context (:tx-data-context ctx')})
```

Comment:
Bug fix: Performing the `:override` transaction step would strip the existing `:tx-data-context` from the transaction `ctx`.